Cubical/Functions/Embedding.agda

```diff
@@ -132,3 +133,20 @@ isEquiv→hasPropFibers e b = isContr→isProp (equiv-proof e b)
 
 isEquiv→isEmbedding : isEquiv f → isEmbedding f
 isEquiv→isEmbedding e = hasPropFibers→isEmbedding (isEquiv→hasPropFibers e)
+
+abstract
```

Comment:
Why is this abstract?

Reply:
There is some complexity explosion when trying to type check shift, they also note this in the paper I used as a guide for the implementation, so an abstract is needed. I tried to push the abstract as far down as I could, so the definition would compute for somethings, therefore this definition ended up as an abstract. I will see if I think of a better solution.

Reply:
Aha, I think the inefficiency will go away if you find a more direct proof of shift-iso. Things tend to get very inefficient if you convert back and forth between iso/equiv/path

Reply:
I have removed the abstract here, and moved it to where the definition is used, in `shift-iso`. Hopefully I can find a better definition of `shift-iso`.

Reply:
Sounds good. I don't think you need a better definition of shift-iso in this PR, but it would be good to leave a comment about it there.